src/AzureClient/Visualization/CloudJobEncoders.cs

```diff
@@ -47,11 +47,11 @@ internal static class CloudJobExtensions
                     ("Job Status", cloudJob => cloudJob.Status),
                     ("Provider", cloudJob => cloudJob.Details.ProviderId),
                     ("Target", cloudJob => cloudJob.Details.Target),
-                    ("Creation Time", cloudJob => cloudJob.Details.CreationTime.ToDateTime()?.ToString()),
-                    ("Begin Execution Time", cloudJob => cloudJob.Details.BeginExecutionTime.ToDateTime()?.ToString()),
-                    ("End Execution Time", cloudJob => cloudJob.Details.EndExecutionTime.ToDateTime()?.ToString()),
+                    ("Creation Time", cloudJob => cloudJob.Details.CreationTime.ToDateTime()?.ToString() ?? string.Empty),
+                    ("Begin Execution Time", cloudJob => cloudJob.Details.BeginExecutionTime.ToDateTime()?.ToString() ?? string.Empty),
+                    ("End Execution Time", cloudJob => cloudJob.Details.EndExecutionTime.ToDateTime()?.ToString() ?? string.Empty),
                 },
-                Rows = jobsList.OrderByDescending(job => job.Details.CreationTime).ToList()
+                Rows = jobsList.OrderByDescending(job => job.Details.CreationTime).ToList(),
```

Comment:
NIT: Was that intentional?

Reply:
Yes, there was a compiler warning that all properties in an object initializer should end with a trailing comma.